robyn/reloader.py

```diff
@@ -118,14 +118,10 @@ def reload(self):
         self.built_rust_binaries = compile_rust_files(self.directory_path)
 
         if self.config.dev and self.config.file_path is not None:
-            split = self.config.file_path.split("/")
-
-            working_directory = "/".join(split[-2:-1])
-
-            subprocess.call(f"cd {working_directory}", shell=True)
+            module_name = self.config.file_path.split("/")[-2]
```

Comment:
are you sure this will always be the case?

i.e. the module name will always be the second to last name?

Reply:
@sansyrox oh, okay, that makes sense. do you have any suggestions on how to do this?!

here are some solutions from the top of my head:

- can we reload it by running it normally (not a module). but, this is undesirable.

Reply:
hey, my thoughts were, `file_path` contains `sys.argv[0]`. which will be the path to the `__main__.py` file. hence, the second to last of it will be it's directory name. hence the module name. won't this always be the case?

Reply:
@VishnuSanal , I believe that we can make use of `__file__` that is being passed in the Robyn() class